adapters/telegram/src/polling.ts

```diff
@@ -83,7 +83,7 @@ export namespace HttpPolling {
 
   export const Config: Schema<Config> = Schema.object({
     protocol: Schema.const('polling' as const).required(),
-    pollingTimeout: Schema.natural().role('time').default(Time.minute).description('通过长轮询获取更新时请求的超时 (单位为秒)。'),
+    pollingTimeout: Schema.natural().role('time').default(25).description('通过长轮询获取更新时请求的超时 (单位为秒)。'),
```

Comment:
I would prefer using ms as unit, changing code to something like `Math.ceil(pollingTimeout / 1000)`.

This config use `role('time')` which implies a unified time unit.

Reply:
> I would prefer using ms as unit, changing code to something like `Math.ceil(pollingTimeout / 1000)`.
> 
> This config use `role('time')` which implies a unified time unit.

Changed